spec/p2p/messages/pex.md

```diff
@@ -25,7 +25,7 @@ PexRequest is an empty message requesting a list of peers.
 
 ### PexResponse
 
-PexResponse is an list of net addresses provided to a peer to dial.
+PexResponse is a list of net addresses provided to a peer to dial.
```

Comment:
please remove this fix and add it to upstream

changes to comet specific things only make pulling upstream changes more difficult

Reply:
done